lib/bootloader_setup.pm

```diff
@@ -35,17 +36,32 @@ our @EXPORT = qw(
   tianocore_select_bootloader
 );
 
+# prevent grub2 timeout; 'esc' would be cleaner, but grub2-efi falls to the menu then
+sub stop_grub_timeout {
```

Comment:
why does "stop_grub_timeout" confuse people? I plan to make use of that function in all cases where we have 2-3 three lines (not 1 line, when you include the copy-pasted comment) to describe why "up" is pressed. I can partially revert that of course and keep it for later when I use it more.